docs/gettingstarted/index.md

```diff
@@ -1,60 +1,119 @@
 Getting Started with Forge
 ==========================
 
-This is a simple guide to get you from nothing to a basic mod. The rest of this documentation is about where to go from here.
+If you have never made a Forge mod before, this section will provide the minimum amount of information needed to setup a Forge development environment. The rest of the documentation is about where to go from here.
+
+Prerequisites
+-------------
+
+* An installation of the Java 17 Development Kit (JDK) and 64-bit Java Virtual Machine (JVM). Forge recommends and officially supports [Eclipse Temurin][jdk].
+
+!!! warning
+    Using a 32-bit JVM will cause some problems when using [ForgeGradle]. Make sure your computer supports a 64-bit architecture along with an associated JVM.
```

Comment:
Might be better to reword this to something along the lines of:
> Make sure you're using a 64-bit JVM. One way of checking is to run `java -version` in a terminal

Pretty much all computers used for MC dev these days have a 64bit architecture. JEP 449 proposes dropping 32bit Windows support in Java, Ubuntu and macOS dropped 32bit support years ago and Microsoft dropped it in Win11.